### DISCUSSION TITLE
Request: Is it possible to not send nginx access.log to dev/nul

### DISCUSSION THREAD

**Discussion by airdogvan** [nginx-proxy/nginx-proxy#2199] [Q&A] [answered]
So I can mount that file and have fail2ban keep an eye.

**Comment by SchoNie** [comment, marked as answer]
Yes, nginx access logs already are written to a file (`/var/log/nginx/access.log`) but inside the container. You could mount this access.log to a volume or host folder, and then expose that to fail2ban.

See: [#2190](https://github.com/nginx-proxy/nginx-proxy/discussions/2190#discussioncomment-5292962)

**Reply by buchdag** [reply]
Alternatively to @SchoNie solution, you can configure Docker to use a different [logging driver](https://docs.docker.com/config/containers/logging/configure/#supported-logging-drivers) (`syslogd` for instance) for the containers whose logs you want analysed by fail2ban.

**Comment by airdogvan** [comment]
Thanks for the trouble.  I thought I searched and didn't find anything.  Must have searched with wrong terms...
